app/views/registrations/_form.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= form_for(@registration ||= Registration.new, remote: true) do |f| %>
```

Comment:
Default assignation logic shouldn't stay in a view, I think it fits better inside `registration_controller`.